README.md

```diff
@@ -1,6 +1,10 @@
 Commissaire
 ===========
-The lightweight REST API for Node Management.
+Commissaire is a lightweight REST interface for maintaining network hosts in a cluster through Ansible.
+
```

Comment:
I'm hesitant to use the word "manage" alone because it's vague and I think might cause confusion about Commissaire's role versus a cluster orchestration system.

Maybe "for performing *system management* tasks on network hosts in a cluster"?